examples/tensor/Jamfile

```diff
@@ -0,0 +1,23 @@
+# Boost.uBLAS
+#
+# Copyright (c) 2018 Cem Bassoy
+#
+# Use, modification and distribution is subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+
+# Project settings
+project boost-ublas-tensor-example
+    : requirements
+       <define>BOOST_UBLAS_NO_EXCEPTIONS
+       <toolset>vacpp:<define>"BOOST_UBLAS_NO_ELEMENT_PROXIES"
+       <toolset>gcc:<cxxflags>"-Wall -pedantic -Wextra -std=c++17"
+       <toolset>gcc:<cxxflags>"-Wno-unknown-pragmas"
+       <toolset>msvc:<cxxflags>"/W4" # == all
+    ;
+
+exe example_construction_access : example_construction_access.cpp ;
```

Comment:
Just for terseness, I would suggest not to repeat the word `example` in the example source files (and executable names). The user knows he is looking at example code in this directory. :-)

Reply:
the user might forget that. you never know at what time you program :-)

Reply:
this has been fixed now.